pkg/ccl/backupccl/backup_planning.go

```diff
@@ -67,6 +67,7 @@ const (
 	localityURLParam          = "COCKROACH_LOCALITY"
 	defaultLocalityValue      = "default"
 	backupOptDebugMetadataSST = "debug_dump_metadata_sst"
+	backupOptEncDir           = "encryption_dir"
```

Comment:
I noted this in the original slack thread too, but for the sake of discussion what if we made this more general than just `encryption_dir`. The only time we need users to specify this additional path to the base backup is when `SHOW BACKUP` is being pointed to an incremental. What if we used:

`SHOW BACKUP x WITH KMS = '…' INCREMENTAL FROM '<base_backup>'` instead?

Reply:
resolved in slack. keeping original approach.